lib/internal/repl/utils.js

```diff
@@ -87,7 +89,98 @@ function isRecoverableError(e, code) {
   }
 }
 
+// Appends the preview of the result
+// to the tty.
+function appendPreview(repl, result, cursorTo, clearScreenDown) {
```

Comment:
```suggestion
function appendPreview(repl, preview, cursorTo, clearScreenDown) {
```
maybe? To make it a little clearer.